scripts/intersections.py

```diff
@@ -144,4 +145,6 @@ def multi_layers(iso3, layer_1, layer_2, cell_generation):
 
 if __name__ == '__main__':
 
-    multi_layers('KEN', 'intersection', 'coverage', 'GSM')
\ No newline at end of file
+    multi_layers('KEN', 'population', 'river_flood', 'GSM')
```

Comment:
This isn't clear. The function returns 'intersection', but it's not clear what you do with this/where you write it?

Reply:
The interesection fucntion, intersects two geodataframes and saves the resulting shapefile into a folder.
I changed repository from my end. It even shows on the repo landing page.
I have plotted the voronoi polygons on Q-GIS